tests/test_types/reference/AB_model.ini

```diff
@@ -0,0 +1,144 @@
+[Meta]
+author = Max J. Hoffmann
+email = mjhoffmann@gmail.com
+model_name = AB_no_diffusion
+model_dimension = 2
+debug = 0
+
+[SpeciesList]
+default_species = empty
+
+[Species A]
+representation = Atoms('O')
+color = 
+tags = 
+
+[Species B]
+representation = Atoms('CO', [[0,0,0],[0,0,1.2]])
+color = 
+tags = 
+
+[Species empty]
+representation = 
+color = 
+tags = 
+
+[Parameter A]
+value = 1.552e-19
+adjustable = False
+min = 0.0
+max = 0.0
+scale = linear
+
+[Parameter E_bind_CO]
+value = -1.9
+adjustable = False
+min = 0.0
+max = 0.0
+scale = linear
+
+[Parameter E_bind_O2]
+value = -2.138
+adjustable = False
+min = 0.0
+max = 0.0
+scale = linear
+
+[Parameter E_react]
+value = 0.9
+adjustable = False
+min = 0.0
+max = 0.0
+scale = linear
+
+[Parameter T]
+value = 600
+adjustable = False
+min = 0.0
+max = 0.0
+scale = linear
+
+[Parameter p_COgas]
+value = 1.0
+adjustable = True
+min = 1e-13
+max = 100.0
+scale = log
+
+[Parameter p_O2gas]
+value = 1.0
+adjustable = True
+min = 1e-13
+max = 100.0
+scale = log
+
+[Lattice]
+cell_size = 1.0 0.0 0.0 0.0 1.0 0.0 0.0 0.0 1.0
+default_layer = default
+substrate_layer = default
+representation = 
+
+[Layer default]
+color = #ffffff
+site a = (0.0, 0.0, 0.0); default_species; 
+
+[Process AB_react_down]
+rate_constant = 1/(beta*h)*exp(-beta*E_react*eV)
+otf_rate = None
```

Comment:
This is being interpreted as a `"None"` string by the importer... I will fix this, to catch the string and pass as a built-in `None`.
May be we should also explcitely prohibit variables called `"None"` across all the code?
Related: Behavior up to know has been to not print anything in the xml file when `process.otf_rate` evaluates to false. I did this to ensure backwards-compatibility (i.e. continue exporting the same .xml when you do not define otf_rate). I do not see a big compelling reason to change this right now.